go/pkg/services/log_collector/log_collector_test.go

```diff
@@ -0,0 +1,205 @@
+// Copyright 2018 Iguazio
```

Comment:
Tests are good :)
I'd add some unit tests for the functions that query kubernetes with mocked k8s responses